packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/exchange/exchange-renew.routing.js

```diff
@@ -1,4 +1,4 @@
-import ExchangeModel from '@ovh-ux/manager-exchange/src/Exchange.class'; // eslint-disable-line import/extensions
+import { Exchange as ExchangeModel } from '@ovh-ux/manager-exchange'; // eslint-disable-line import/extensions
```

Comment:
I think the ESLint comment could be removed:
```suggestion
import { Exchange as ExchangeModel } from '@ovh-ux/manager-exchange';
```

Reply:
I think it was done due to the previous import but I forgot to remove it